### PR TITLE
Add WildFly Swarm stack to default assembly

### PR DESCRIPTION
**Pull request by dharmit**
### What does this PR do?

Adds CentOS based WildFly Swarm stack to default assembly

#### Changelog

Add WildFly Swarm stack to default assembly

#### Release Notes

You can now build WildFly Swarm based applications on CentOS Linux by easily creating a workspace from the WildFly Swarm stack.

#### Docs PR
TODO Create tutorial. 

**Recipe PR**: [che-dockerfiles/pull/79](https://github.com/eclipse/che-dockerfiles/pull/79)

### CODE REVIEW

File: ide/che-core-ide-stacks/src/main/resources/stacks.json [hunk 1]
```diff
@@ -2135,5 +2135,81 @@
...
+      "name": "CentOS",
+      "version": "---"
+    },
+    {
+      "name": "JDK",
+      "version": "1.8.0_45"
+    },
+    {
+      "name": "Maven",
+      "version": "3.2.2"
```
Comment (kenfinnigan) [inline ide/che-core-ide-stacks/src/main/resources/stacks.json]: @dharmit We need Maven 3.2.5 and above for WF Swarm

Comment (dharmit) [reply ide/che-core-ide-stacks/src/main/resources/stacks.json]: @kenfinnigan Here's the output from inside the workspace:

```bash
$ mvn -v                                                                                                                                                                                    
Apache Maven 3.3.9 (Red Hat 3.3.9-2.8)                                                                                                                                                                           
Maven home: /opt/rh/rh-maven33/root/usr/share/maven                                                                                                                                                              
...
```
I'll modify the `stacks.json` file. Thanks for the pointer.

File: ide/che-core-ide-stacks/src/main/resources/stacks.json [hunk 2]
```diff
@@ -2135,5 +2135,81 @@
...
         "mediaType": "image/svg+xml"
     }
-  }
+  },
+  {
+  "name": "CentOS WildFly Swarm",
+  "id": "WildFly Swarm",
+  "creator": "Dharmit Shah",
+  "description": "Eclipse WildFly Swarm Stack on CentOS.",
+  "scope": "general",
```
Comment (l0rd) [inline ide/che-core-ide-stacks/src/main/resources/stacks.json]: Setting scope to general will show the stack in the restricted list of "Ready-to-go" stacks. Are you sure you want to do this?

File: ide/che-core-ide-stacks/src/main/resources/stacks.json [hunk 3]
```diff
@@ -2385,4 +2385,80 @@
       "mediaType": "image/svg+xml"
     }
   }
+=======
```
Comment (ghost) [inline ide/che-core-ide-stacks/src/main/resources/stacks.json]: Shouldn't it be { ? Your JSON is malformed this way.

Comment (dharmit) [reply ide/che-core-ide-stacks/src/main/resources/stacks.json]: @eivantsov absolutely! corrected it now. Thank you!

**Review by ghost — APPROVED** [review]

**Review by l0rd — APPROVED** [review]

**Comment by l0rd** [comment]
@dharmit we are going to merge that as soon as Che v5.8 has been released

**Comment by JamesDrummond** [comment]
Please create an issue in `eclipse/che-docs` to add tutorial for this stack. Added TODO above in PR description.

**Comment by l0rd** [comment]
@JamesDrummond done https://github.com/eclipse/che-docs/issues/209

**Pull request merged**